packages/core/src/codewhisperer/service/transformByQ/transformApiHandler.ts

```diff
@@ -178,16 +182,12 @@ export async function resumeTransformationJob(jobId: string, userActionStatus: T
             result: MetadataResult.Fail,
             reason: 'ResumeTransformationFailed',
         })
-        throw new Error('Resume transformation job failed')
+        throw new Error(errorMessage)
     }
 }
 
 export async function stopJob(jobId: string) {
     if (!jobId) {
-        throw new Error('Job ID is empty')
```

Comment:
Don't throw an error since that gets surfaced to customer; just avoid calling `stopJob` if no `jobId` has been generated yet.